stdlib/tkinter/__init__.pyi

```diff
@@ -667,12 +702,113 @@ class Wm:
         self, minNumer: None = None, minDenom: None = None, maxNumer: None = None, maxDenom: None = None
     ) -> tuple[int, int, int, int] | None: ...
     aspect = wm_aspect
+    if sys.version_info >= (3, 13):
+        @overload
+        def wm_attributes(self, *, return_python_dict: Literal[False] = False) -> tuple[Any, ...]: ...
+        @overload
+        def wm_attributes(self, *, return_python_dict: Literal[True]) -> _WmAttributes: ...
+
+    else:
+        @overload
+        def wm_attributes(self) -> tuple[Any, ...]: ...
+
     @overload
-    def wm_attributes(self) -> tuple[Any, ...]: ...
+    def wm_attributes(self, option: Literal["alpha"], /) -> float: ...
+    @overload
+    def wm_attributes(self, option: Literal["fullscreen"], /) -> bool: ...
+    @overload
+    def wm_attributes(self, option: Literal["topmost"], /) -> bool: ...
+    if sys.platform == "darwin":
+        @overload
+        def wm_attributes(self, option: Literal["modified"], /) -> bool: ...
+        @overload
+        def wm_attributes(self, option: Literal["notify"], /) -> bool: ...
+        @overload
+        def wm_attributes(self, option: Literal["titlepath"], /) -> str: ...
+        @overload
+        def wm_attributes(self, option: Literal["transparent"], /) -> bool: ...
+        @overload
+        def wm_attributes(self, option: Literal["type"], /) -> str: ...
+    elif sys.platform == "win32":
+        @overload
+        def wm_attributes(self, option: Literal["transparentcolor"], /) -> str: ...
+        @overload
+        def wm_attributes(self, option: Literal["disabled"], /) -> bool: ...
+        @overload
+        def wm_attributes(self, option: Literal["toolwindow"], /) -> bool: ...
+    else:
+        # X11
+        @overload
+        def wm_attributes(self, option: Literal["zoomed"], /) -> bool: ...
+        @overload
+        def wm_attributes(self, option: Literal["type"], /) -> str: ...
```

Comment:
These don't seem correct. This form wants the option with a dash in front:

```python
>>> import tkinter
>>> r = tkinter.Tk()
>>> r.wm_attributes("zoomed")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.11/tkinter/__init__.py", line 2032, in wm_attributes
    return self.tk.call(args)
           ^^^^^^^^^^^^^^^^^^
_tkinter.TclError: bad attribute "zoomed": must be -alpha, -topmost, -zoomed, -fullscreen, or -type
>>> r.wm_attributes("-zoomed")
0
```

Reply:
Looks like 3.13 supports getting `"alpha"` as well as `"-alpha"`. Setting requires `"-alpha"` on both 3.13 and 3.12. I also noticed that setting `r.wm_attributes("-alpha", 1.0)` actually returns the empty string, not None. `r.wm_attributes(alpha=1.0)` does return None though.